models/turbine_models/tests/stateless_llama_test.py

```diff
@@ -30,6 +35,22 @@
 Be concise. You are a helpful, respectful and honest assistant. If a question does not make any sense, or is not factually coherent, explain why instead of answering something not correct. If you don't know the answer to a question, please don't share false information. <</SYS>> hi what are you? [/INST]
 """
 
+tokenizer = AutoTokenizer.from_pretrained(
```

Comment:
Note that this does a large download and setup during module loading (i.e. which includes enumerating which tests to run). Such things should be scoped to a setup task).

(I know it was like this so just trying to point out)

Reply:
Ah I see, done